posts/2024-06-13-Rust-1.79.0.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: "Announcing Rust 1.79.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.79.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.79.0 with:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.79.0](https://doc.rust-lang.org/nightly/releases.html#version-1790-2024-06-13).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.79.0 stable
+
+### Inline `const` expressions
+
+`const { ... }` blocks are now stable in expression position, permitting
+explicitly entering a const context without requiring extra declarations (e.g.,
+defining `const` items or associated constants on a trait).
+
+Unlike const items (`const ITEM: ... = ...`), inline consts are able to make
+use of in-scope generics, making them particularly useful for inline code snippets. For example, a pattern like:
```

Comment:
This paragraph says “in-scope generics”, but the example code immediately below is not an example of using in-scope generics; it is an example of using type inference.

These are both important things for this class of use case, so I think a good fix would be to just mention both:

```suggestion
use of in-scope generics, and have their type inferred rather than written explicitly, making them particularly useful for inline code snippets. For example, a pattern like:
```